### PR TITLE
Feat/login page

### PR DESCRIPTION
**Pull request by MobileAveo**

### CODE REVIEW

File: public/assets/images/home/temp/jarvischoice.png
Comment (NguyenNL) [inline public/assets/images/home/temp/jarvischoice.png:1]: Do we use this image ? if not use, please remove

File: public/assets/images/home/temp/location.png
Comment (NguyenNL) [inline public/assets/images/home/temp/location.png:1]: Do we use this image ? if not use, please remove

**Review by NguyenNL — CHANGES_REQUESTED** [review]

File: public/assets/images/home/temp/jarvischoice.png
Comment (MobileAveo) [reply public/assets/images/home/temp/jarvischoice.png:1]: yes we are using

File: public/assets/images/home/temp/location.png
Comment (MobileAveo) [reply public/assets/images/home/temp/location.png:1]: yes we are using

File: public/assets/images/home/temp/promotion.png
[the remark this answers is not in the document]
Comment (MobileAveo) [reply public/assets/images/home/temp/promotion.png:1]: yes we are using for now will remove later

File: public/assets/images/home/temp/jarvischoice.png
Comment (NguyenNL) [reply public/assets/images/home/temp/jarvischoice.png:1]: where ?

File: public/assets/images/home/temp/promotion.png
[the remark this answers is not in the document]
Comment (NguyenNL) [reply public/assets/images/home/temp/promotion.png:1]: where ? please share image

File: src/layouts/main/footer.tsx [hunk 1]
```diff
@@ -46,126 +54,149 @@ const LINKS = [
...
+            <Grid xs={8} md={3}>
+              <WhiteLogo sx={{ mb: 3 }} />
+              <Typography
+                variant="body2"
+                sx={{
+                  maxWidth: 270,
+                  mx: { xs: 'auto', md: 'unset' },
+                }}
+              >
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
```
[the remark this answers is not in the document]
Comment (NguyenNL) [reply src/layouts/main/footer.tsx:115]: You need to ask Bella for this text

[the remark this answers is not in the document]
Comment (MobileAveo) [reply src/layouts/main/footer.tsx:115]: okay

File: public/assets/images/home/temp/jarvischoice.png
Comment (MobileAveo) [reply public/assets/images/home/temp/jarvischoice.png:1]: it's in home page

File: public/assets/images/home/temp/location.png
Comment (MobileAveo) [reply public/assets/images/home/temp/location.png:1]: it's in home page

File: next.config.js
Comment (NguyenNL) [inline next.config.js:1]: Please help to revert this file.

File: public/assets/images/home/temp/promotion.png
[the remark this answers is not in the document]
Comment (NguyenNL) [reply public/assets/images/home/temp/promotion.png:1]: We already have api for this ? Did you integrate ?

**Pull request merged**